src/v2/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -44,7 +45,11 @@ const AuctionDetails: React.FC<AuctionDetailsProps> = ({ sale, me }) => {
       </GridColumns>
 
       <Spacer my={4} />
-
+      <Flex alignItems="center" justifyContent="space-between">
+        <Flex alignItems="center">
+          <SaleDetailTimerFragmentContainer sale={sale} />
```

Comment:
Is it right to render this on all sales (even w/o cascading)? It feels like it might be odd to render this above the below `AuctionDetailsStartTimeQueryRenderer` (two timers)?

Relatedly- pls update the PR description w/ screenshots of the changes!